library/board.py

```diff
@@ -178,5 +125,7 @@ def redraw_subgrid(
             "7": (12, 1),
             "8": (24, 1),
         }
+        self.redraw_gridlines(term, start_coords[number])
```

Comment:
Should this take a color parameter? It sounds like we want the subgrids to have certain colors to show certain actions.

Reply:
Added a color parameter to the redraw_subgrid method that gets passed to redraw_gridlines
```
            "8": (24, 1),
        }
        self.redraw_gridlines(term, start_coords[number], color)
        self.redraw_gamestate(term, subgrid, start_coords[number])
```